src/status_im/contexts/chat/messenger/composer/handlers.cljs

```diff
@@ -164,8 +160,3 @@
       (let [{:keys [start end text-input-handle]} @selection-event]
         (selection/update-selection text-input-handle start end)
         (reset! selection-event nil)))))
-
-(defn layout
-  [event state blur-height]
-  (when (utils/update-blur-height? event state blur-height)
-    (reanimated/set-shared-value blur-height (oops/oget event "nativeEvent.layout.height"))))
```

Comment:
Do we not need this anymore?

Reply:
Composer is only blurred when it is collapsed and empty, not sure why we animated blur height in first place. But yes we don't this anymore